packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -1133,11 +1141,11 @@ export class ComponentDecoratorHandler implements
 
   private _parseTemplate(
       template: TemplateDeclaration, sourceStr: string, sourceParseRange: LexerRange|null,
-      escapedString: boolean): ParsedComponentTemplate {
+      escapedString: boolean, sourceMapUrl: string|null): ParsedComponentTemplate {
     // We always normalize line endings if the template has been escaped (i.e. is inline).
     const i18nNormalizeLineEndingsInICUs = escapedString || this.i18nNormalizeLineEndingsInICUs;
 
-    const parsedTemplate = parseTemplate(sourceStr, template.sourceMapUrl, {
+    const parsedTemplate = parseTemplate(sourceStr, template.potentialSourceMapUrl, {
```

Comment:
Should this be:

```suggestion
    const parsedTemplate = parseTemplate(sourceStr, sourceMapUrl ?? '', {
```

similar to the other calls to `parseTemplate()`?

Reply:
Yes indeed - fixed.